common/src/config.rs

```diff
@@ -0,0 +1,119 @@
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct BootloaderConfigFile {
```

Comment:
Where is this type serialized? I can't find it.

Reply:
Or is that part not yet written?

Reply:
Thanks, my idea was for serializing from the kernel, passing that file to disk image creator and then de-serializing at runtime.

Reply:
Makes sense